packages/features/ee/organizations/lib/orgDomains.ts

```diff
@@ -23,12 +23,21 @@ export function getOrgSlug(hostname: string) {
   return null;
 }
 
-export function orgDomainConfig(hostname: string) {
+export function orgDomainConfig(hostname: string, fallback?: string | string[]) {
```

Comment:
We now provide a possible fallback in case host is not a valid subdomain, this is to take into account accessing through `/org/orgSlug`